### ISSUE TITLE
[Feature request] hide input 

### ISSUE THREAD

**Issue by daaa57150** [1j01/simple-console#1]
Hi,

It would be cool to have the ability to hide the input so that it just acts as an output console. I used your library and hid the input with css, it does the job but a function/option in the API to do so would be really cool.

**Comment by 1j01** [comment]
Done in https://github.com/1j01/simple-console/commit/520f0e94b191877458b3448210bc9301ad2b9df2. Can I ask what you're using this for? Just curious.

**Comment by daaa57150** [comment]
Yeah, it's for a technical POC dealing with video transcoding. There are many jobs launched in parallel and it can be hard to follow what's going on; so that's where your simple console comes in. All the important messages are written in there for the users to see ("done this", "done that"...) and all the details are still in the developer console for me. 

Thanks for the fast response.

**Comment by 1j01** [comment]
Cool.
